testsuite/shared/src/main/java/org/jboss/as/test/integration/transactions/ManagementUtil.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright (c) 2016, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+
+package org.jboss.as.test.integration.transactions;
+
+import java.io.IOException;
+
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.*;
+
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.as.controller.client.ModelControllerClient;
+import org.jboss.as.test.integration.management.util.MgmtOperationException;
+import org.jboss.dmr.ModelNode;
+import org.jboss.logging.Logger;
+
+/**
+ * Utility class for management operations
+ *
+ * @author Ondra Chaloupka <ochaloup@redhat.com>
+ */
+public final class ManagementUtil {
+    private static final Logger log = Logger.getLogger(ManagementUtil.class);
+
+    public static ModelNode readAttribute(ModelNode address, String name, ManagementClient mgmtClient) throws IOException, MgmtOperationException {
+        ModelNode operation = new ModelNode();
```

Comment:
This can already be done with 
`import org.jboss.as.controller.operations.common.Util;
...
        ModelNode op = Util.getReadAttributeOperation(PathAddress.pathAddress(address), name);
`
include-defaults defaults to true anyway